src/Symfony/Component/Security/Tests/Acl/Voter/AclVoterTest.php

```diff
@@ -27,7 +27,7 @@ class AclVoterTest extends \PHPUnit_Framework_TestCase
      */
     public function testSupportsAttribute($attribute, $supported)
     {
-        list($voter, , $permissionMap, ,) = $this->getVoter(true, false);
+        list($voter, , $permissionMap, , ) = $this->getVoter(true, false);
```

Comment:
I would prefer to remove those trailing commas instead of adding those "weird-looking" whitespaces.